VTCare/client/src/types.ts

```diff
@@ -22,6 +22,17 @@ export interface Provider{
   availabilitySchedule: AppointmentSlot[];
 }
 
+export interface Patient{
```

Comment:
The patient object if you look at the backend code contains "history" object of type List<PatientVisitHistory> as well. Please add the history interface and its property here.

Patient also contains an insurance object if you refer to the backend code.